pkg/oauth/registry/oauthaccesstoken/etcd/etcd.go

```diff
@@ -52,6 +56,22 @@ func NewREST(s storage.Interface) *REST {
 
 	store.CreateStrategy = oauthaccesstoken.Strategy
 
+	if len(backends) > 0 {
+		// Build identical stores that talk to a single etcd, so we can verify the token is distributed after creation
+		watchers := []rest.Watcher{}
+		for i := range backends {
+			watcher := *store
+			watcher.Storage = backends[i]
+			watchers = append(watchers, &watcher)
+		}
+		// Observe the cluster for the particular resource version, requiring at least one backend to succeed
+		observer := observe.NewClusterObserver(s.Versioner(), watchers, 1)
```

Comment:
I'm hesitant to introduce new failure cases without letting this bake longer. Could revisit after 1.1.1